jetstream/leafnodes.md

```diff
@@ -0,0 +1,688 @@
+             Replicas: 1
+       Discard Policy: Old
+     Duplicate Window: 2m0s
+     Maximum Messages: unlimited
+        Maximum Bytes: unlimited
+          Maximum Age: 0.00s
+ Maximum Message Size: unlimited
+    Maximum Consumers: unlimited
+              Sources: test, API Prefix: $JS.leaf.API, Delivery Prefix:
+
+
+State:
+
+             Messages: 0
+                Bytes: 0 B
+             FirstSeq: 0
+              LastSeq: 0
+     Active Consumers: 0
+```
+
+`source` as well as `mirror` take a copy of the messages.
+Once copied, accessing the data is independent of the leaf node connection being online.
+Copying this way also avoids having to run a dedicated program of your own. 
+This is the recommended way to exchange persistent data across domains.
+
+```bash
+> nats  --server nats://acc:acc@localhost:4222 stream report --js-domain hub
+Obtaining Stream stats
+
+╭──────────────────────────────────────────────────────────────────────────────────────────╮
+│                                      Stream Report                                       │
+├─────────────────────┬─────────┬───────────┬──────────┬───────┬──────┬─────────┬──────────┤
+│ Stream              │ Storage │ Consumers │ Messages │ Bytes │ Lost │ Deleted │ Replicas │
+├─────────────────────┼─────────┼───────────┼──────────┼───────┼──────┼─────────┼──────────┤
+│ backup-test-leaf    │ File    │ 0         │ 1        │ 45 B  │ 0    │ 0       │          │
+│ test                │ File    │ 0         │ 1        │ 45 B  │ 0    │ 0       │          │
+│ aggregate-test-leaf │ File    │ 0         │ 1        │ 98 B  │ 0    │ 0       │          │
+╰─────────────────────┴─────────┴───────────┴──────────┴───────┴──────┴─────────┴──────────╯
+
+╭────────────────────────────────────────────────────────────────────────────────────╮
+│                                 Replication Report                                 │
+├─────────────────────┬────────┬──────────────┬───────────────┬────────┬─────┬───────┤
+│ Stream              │ Kind   │ API Prefix   │ Source Stream │ Active │ Lag │ Error │
+├─────────────────────┼────────┼──────────────┼───────────────┼────────┼─────┼───────┤
+│ backup-test-leaf    │ Mirror │ $JS.leaf.API │ test          │ 0.21s  │ 0   │       │
+│ aggregate-test-leaf │ Source │ $JS.leaf.API │ test          │ 1.23s  │ 0   │       │
+╰─────────────────────┴────────┴──────────────┴───────────────┴────────┴─────┴───────╯
+```
+
+#### Cross account & domain import 
+
+All of the above happened in the same account.
+To share domain access across accounts the `account.conf` from above needs to be modified and the server restarted or reloaded.
+This example exports the consumer API as well as a delivery subject which is used by the internal push consumer created by `source` and `mirror`.
+
+In support of an example on how to share a durable consumer for client access across domains and accounts, the `NEXT` and `ACK` API are exported as well.
```

Comment:
The "in support of an example on how to share a durable consumer for client across domains and accounts" - These are all prefixed to the same prefix (er domain), right? Not sure why the distinction is needed.

Reply:
changed it to in support of another example.
basically I have one setup, then show first example (source/mirror) then examples for clients